.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: ruby
-cache: bundler
+branches:
```

Comment:
This will limit travis branch builds to only run on develop and master. Travis Pull builds will still run on every commit after you opened a pull request. This configuration is recommended by travis to reduce the amount of builds. Otherwise you always build everythink twice as soon as you open a pull request. It's also a waste of resources to run travis on every commit someone did, even if he is just testing around and have no intention to merge this code soon.